echopype/tests/calibrate/test_calibrate.py

```diff
@@ -244,7 +245,7 @@ def test_compute_Sv_ek80_pc_echoview(ek80_path):
     df_real = df.loc[df['Component'] == ' Real', :].iloc[:, 14:]
 
     # Compare only values for range > 0: difference is surprisingly large
-    range_meter = cal_obj.range_meter.isel(frequency=0, ping_time=0).values
+    range_meter = cal_obj.range_meter.isel(channel=1, ping_time=0).values
```

Comment:
Can we change this to use `.sel(channel=SOME_CHANNEL)` so that it is clear which channel we are comparing?
Based on the change made here, it seems that the frequency values are not in a monotonic (or increasing) order when aligned with `channel`, so here the change was to `channel=1` instead of `frequency=0`.